src/community/develop-a-component-or-pattern/index.md.njk

```diff
@@ -32,7 +32,7 @@ If you’re happy to go ahead, the team will assign you to the issue as a contri
 
 While you’re working on your contribution, the Design System team will arrange a weekly catch up to find out how you’re getting on and if you need any help. 
 
-Find examples of the component or pattern already in use. The best way to do this is to ask the government design community. Use the [Design Slack channel](https://ukgovernmentdigital.slack.com/messages/C062GAGLW/) and the [digital service designers mailing list](https://groups.google.com/a/digital.cabinet-office.gov.uk/forum/?hl=en-GB#!forum/digital-service-designers).
+Find examples of the component or pattern already in use. The best way to do this is to ask the government design community. Use the [#design channel on cross-government Slack](https://ukgovernmentdigital.slack.com/app_redirect?channel=design) and the [digital service designers mailing list](https://groups.google.com/a/digital.cabinet-office.gov.uk/forum/?hl=en-GB#!forum/digital-service-designers).
```

Comment:
```suggestion
Find examples of the component or pattern already in use. The best way to do this is to ask the government design community on the [digital service designers mailing list](https://groups.google.com/a/digital.cabinet-office.gov.uk/forum/?hl=en-GB#!forum/digital-service-designers) or the [#design channel on cross-government Slack](https://ukgovernmentdigital.slack.com/app_redirect?channel=design).
```

Reply:
I've suggested reversing this, as there was a slight risk someone might read as 'the #design channel on... the digital service designers mailing list', which might be confusing.